tests/caasp/first_boot.pm

```diff
@@ -27,6 +27,12 @@ sub run {
     }
 
     microos_login;
+    if (check_var('DISTRI', 'kubic')) {
+        assert_script_run("for repo in \$(zypper lr -U | grep download.opensuse.org | awk '{ print \$1 }'); do zypper mr -d \$repo; done");
```

Comment:
You can use `zypper mr -da` to disable all repos. Do you expect other than download.opensuse.org repositories?

Reply:
I have no idea what is expected, that's why I disabled just the ones that @sysrich mentioned. I know that there's also the DVD repo. So, should I disable them all or not?

Reply:
I'm happy either way

Reply:
Ok let me try to disable them all as @kravciak suggested. This is the new verification run:
http://skyrim.qam.suse.de/tests/3352#step/kubeadm/8